app/controllers/data_access_controller.rb

```diff
@@ -192,6 +197,17 @@ def options
 
   private
 
+  def echo_form_outdated?(access_config, methods)
+    return true if access_config.nil? || access_config.echoform_digest.nil? || methods.nil?
```

Comment:
It will consider the (non-exist) echoform out dated and won't try to load from AccessConfigure.